docker-compose.yml

```diff
@@ -16,9 +24,68 @@ services:
     depends_on:
       - db
     restart: unless-stopped
+    networks:
+      - monitoring
```

Comment:
I think the network name should be `periodum`

Reply:
It's a good idea, "monitoring" was a forgotten one to change.
Done with the last commit.